snmp/datadog_checks/snmp/data/default_profiles/cradlepoint.yaml

```diff
@@ -0,0 +1,118 @@
+extends:
+  - _base.yaml
+  - _generic-if.yaml
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "cradlepoint"
+sysobjectid:
+  - 1.3.6.1.4.1.20992.2.*
+  - 1.3.6.1.4.1.20992.2.1
+  - 1.3.6.1.4.1.20992.2.2
+  - 1.3.6.1.4.1.20992.2.3
+  - 1.3.6.1.4.1.20992.2.4
+  - 1.3.6.1.4.1.20992.2.5
+  - 1.3.6.1.4.1.20992.2.6
+  - 1.3.6.1.4.1.20992.2.7
+  - 1.3.6.1.4.1.20992.2.8
+  - 1.3.6.1.4.1.20992.2.9
+  - 1.3.6.1.4.1.20992.2.10
+  - 1.3.6.1.4.1.20992.2.11
+  - 1.3.6.1.4.1.20992.2.12
+  - 1.3.6.1.4.1.20992.2.13
+  - 1.3.6.1.4.1.20992.2.14
+  - 1.3.6.1.4.1.20992.2.15
+  - 1.3.6.1.4.1.20992.2.16
+  - 1.3.6.1.4.1.20992.2.17
+  - 1.3.6.1.4.1.20992.2.18
+  - 1.3.6.1.4.1.20992.2.19
+  - 1.3.6.1.4.1.20992.2.20
+  - 1.3.6.1.4.1.20992.2.21
+  - 1.3.6.1.4.1.20992.2.22
+  - 1.3.6.1.4.1.20992.2.23
+  - 1.3.6.1.4.1.20992.2.24
+  - 1.3.6.1.4.1.20992.2.25
+  - 1.3.6.1.4.1.20992.2.26
+  - 1.3.6.1.4.1.20992.2.27
+  - 1.3.6.1.4.1.20992.2.28
+  - 1.3.6.1.4.1.20992.2.29
+  - 1.3.6.1.4.1.20992.2.30
+  - 1.3.6.1.4.1.20992.2.31
+  - 1.3.6.1.4.1.20992.2.32
+  - 1.3.6.1.4.1.20992.2.33
+  - 1.3.6.1.4.1.20992.2.34
+  - 1.3.6.1.4.1.20992.2.35
+  - 1.3.6.1.4.1.20992.2.36
+  - 1.3.6.1.4.1.20992.2.37
+  - 1.3.6.1.4.1.20992.2.38
+  - 1.3.6.1.4.1.20992.2.39
+  - 1.3.6.1.4.1.20992.2.40
+  - 1.3.6.1.4.1.20992.2.41
+  - 1.3.6.1.4.1.20992.2.42
+  - 1.3.6.1.4.1.20992.2.43
+  - 1.3.6.1.4.1.20992.2.44
+  - 1.3.6.1.4.1.20992.2.45
+  - 1.3.6.1.4.1.20992.2.46
```

Comment:
🥜 since `1.3.6.1.4.1.20992.2.*` covers all the other sysoids, there's no need to specify them unless we want to mention the models
```suggestion
  - 1.3.6.1.4.1.20992.2.*     # Cradlepoint
  - 1.3.6.1.4.1.20992.2.1     # Cradlepoint mbr1000
  - 1.3.6.1.4.1.20992.2.2     # Cradlepoint ctr500
  - 1.3.6.1.4.1.20992.2.3     # Cradlepoint mbr800
  - 1.3.6.1.4.1.20992.2.4     # Cradlepoint mbr1100
  - 1.3.6.1.4.1.20992.2.5     # Cradlepoint mbr1200
  - 1.3.6.1.4.1.20992.2.6     # Cradlepoint mbr900
  - 1.3.6.1.4.1.20992.2.7     # Cradlepoint cba250
  - 1.3.6.1.4.1.20992.2.8     # Cradlepoint cba750
  - 1.3.6.1.4.1.20992.2.9     # Cradlepoint cx111
  - 1.3.6.1.4.1.20992.2.10    # Cradlepoint mbr1400
  - 1.3.6.1.4.1.20992.2.11    # Cradlepoint mbr1200b
  - 1.3.6.1.4.1.20992.2.12    # Cradlepoint cbr400
  - 1.3.6.1.4.1.20992.2.13    # Cradlepoint cbr450
  - 1.3.6.1.4.1.20992.2.14    # Cradlepoint ibr600
  - 1.3.6.1.4.1.20992.2.15    # Cradlepoint ibr650
  - 1.3.6.1.4.1.20992.2.16    # Cradlepoint mbr1400v2
  - 1.3.6.1.4.1.20992.2.17    # Cradlepoint cba750b
  - 1.3.6.1.4.1.20992.2.18    # Cradlepoint aer2100
  - 1.3.6.1.4.1.20992.2.19    # Cradlepoint ibr1150
  - 1.3.6.1.4.1.20992.2.20    # Cradlepoint ibr1100
  - 1.3.6.1.4.1.20992.2.21    # Cradlepoint cba850
  - 1.3.6.1.4.1.20992.2.22    # Cradlepoint ibr350
  - 1.3.6.1.4.1.20992.2.23    # Cradlepoint aer3100
  - 1.3.6.1.4.1.20992.2.24    # Cradlepoint aer1600
  - 1.3.6.1.4.1.20992.2.25    # Cradlepoint ibr650b
  - 1.3.6.1.4.1.20992.2.26    # Cradlepoint aer3150
  - 1.3.6.1.4.1.20992.2.27    # Cradlepoint aer1650
  - 1.3.6.1.4.1.20992.2.28    # Cradlepoint aer2150
  - 1.3.6.1.4.1.20992.2.29    # Cradlepoint ibr600b
  - 1.3.6.1.4.1.20992.2.30    # Cradlepoint ibr950
  - 1.3.6.1.4.1.20992.2.31    # Cradlepoint ibr900
  - 1.3.6.1.4.1.20992.2.32    # Cradlepoint ibr650c
  - 1.3.6.1.4.1.20992.2.33    # Cradlepoint ibr600c
  - 1.3.6.1.4.1.20992.2.34    # Cradlepoint ibr1700
  - 1.3.6.1.4.1.20992.2.35    # Cradlepoint aer2200
  - 1.3.6.1.4.1.20992.2.36    # Cradlepoint aer2250
  - 1.3.6.1.4.1.20992.2.37    # Cradlepoint ap22
  - 1.3.6.1.4.1.20992.2.38    # Cradlepoint ibr200
  - 1.3.6.1.4.1.20992.2.39    # Cradlepoint ibr250
  - 1.3.6.1.4.1.20992.2.40    # Cradlepoint ibr900-fips
  - 1.3.6.1.4.1.20992.2.41    # Cradlepoint ibr950-fips
  - 1.3.6.1.4.1.20992.2.42    # Cradlepoint ibr1700-fips
  - 1.3.6.1.4.1.20992.2.43    # Cradlepoint aer2200-fips
  - 1.3.6.1.4.1.20992.2.44    # Cradlepoint aer2250-fips
  - 1.3.6.1.4.1.20992.2.45    # Cradlepoint cr4250
  - 1.3.6.1.4.1.20992.2.46    # Cradlepoint cba550
```